org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractLanguageServerTest.xtend

```diff
@@ -217,13 +219,26 @@ abstract class AbstractLanguageServerTest implements Endpoint {
 	}
 	
 	protected def InitializeResult initialize((InitializeParams)=>void initializer, boolean callInitialized) {
+		initialize(initializer, callInitialized, false)
+	}
+	
+	protected def InitializeResult initialize((InitializeParams)=>void initializer, boolean callInitialized, boolean useRootPath) {
 		val params = new InitializeParams => [
 			processId = 1
 			workspaceFolders = #[
 				new WorkspaceFolder(root.toURI.normalize.toUriString, '')
 			]
 		]
 		initializer?.apply(params)
```

Comment:
The initializer should be called after the evaluation of `useRootPath` - otherwise the new default behavior might override previously configured params.